projects/helper/portedTokens.js

```diff
@@ -1527,6 +1527,14 @@ const bittorrentFixMapping = {
   "0x9b5f27f6ea9bbd753ce3793a07cba3c74644330d": {
     coingeckoId: "tether",
     decimals: 18
+  },
+  "0x23181f21dea5936e24163ffaba4ea3b316b57f3c": {
+    coingeckoId: 'bittorrent',
+    decimals: 18,
+  },
+  "TNUC9Qb1rRpS5CbWLmNMxXBjyFoydXjWFR": {
```

Comment:
this mapping is incorrect?

Reply:
ignore the comment, I see that you are compensating for 


    "0xedf53026aea60f8f75fca25f8830b7e2d6200662":
      "tron:TNUC9Qb1rRpS5CbWLmNMxXBjyFoydXjWFR", // TRX